app/components/landing-timeline.js

```diff
@@ -0,0 +1,245 @@
+// https://github.com/EnCiv/undebate-ssp/issues/111
+
+import React from 'react'
+import { createUseStyles } from 'react-jss'
+import {
```

Comment:
There is now an index in svgr so you can import icons from their directly.  But the names don't start with Svg so I wish you would import {SolidTriangleArrow as SvgSolidTriangleArrow, ...}

You can also see all the icons in the new Show Icons story.

Reply:
I changed the import to be from '../svgr' thats all that was needed.